docs/installconfig/configuration/notesclient.md

```diff
@@ -5,10 +5,22 @@ parent: Installing & configuring
 nav_order: 5
 ---
 
+{::options parse_block_html="true" /}
+
 ## Notes client config
 
 Try the KEEP Open API with a sample demo Notes database with your existing HCL Notes Desktop Client.
 
+<div class="panel panel-danger">
+**KEEP on Notes Client**
+{: .panel-heading}
+<div class="panel-body">
+The HCL Domino REST API is designed for use with the HCL Domino server. The ability to deploy it to a Notes client on Windows or macOS is provided for ypur convenience to enable you to evaluate and experiment. You can explore the API without the need to setup a HCL Domino server.
```

Comment:
Please change to -

The ability to deploy it to a Notes client on Windows or macOS is provided for your convenience- to enable you to evaluate and experiment. You can explore the API without setting up a HCL Domino server.